src/controllers/MediaConnectorController.ts

```diff
@@ -158,3 +160,45 @@ export class MediaConnectorController {
         if (deprecatedType === DeprecatedMediaType.collection) return MediaType.collection;
     };
 }
+
+/**
+ * The ConnectorAuthenticationController is responsible for authenticating regarding media connectors.
+ * Methods inside this controller can be called by `window.SDK.mediaConnector.authentication.{method-name}`
+ */
+export class ConnectorAuthenticationController {
+    /**
+     * @ignore
+     */
+    #editorAPI: EditorAPI;
+
+    /**
+     * @ignore
+     */
+    constructor(editorAPI: EditorAPI) {
+        this.#editorAPI = editorAPI;
+    }
+
+    /**
+     * This method sets the token for the 'chili' authentication type.
+     * @param connectorId unique Id of the media connector
+     * @param token token for the chili authentication
+     */
+    setChiliToken = async (connectorId: string, token: string) => {
+        const res = await this.#editorAPI;
+        return res.connectorAuthenticationSetChiliToken(connectorId, token)
+            .then((result) => getEditorResponseData<null>(result));
+    }
+
+    /**
+     * This method sets the http headers for the 'staticKey' authentication type.
+     * @param connectorId unique Id of the media connector
+     * @param headerName name of the header
+     * @param headerValue value of the header
```

Comment:
Same here, please be more descriptive. What is 'Static Key' 
What should the headerValue be? is it a token, is it a random string?,...